src/js/components/uploadDetachedFiles/UploadDetachedFileMeta.jsx

```diff
@@ -315,8 +301,19 @@ export default class UploadDetachedFileMeta extends React.Component {
 						<div className='col-lg-offset-2 col-lg-8 mt-60 mb-60'>
 							<h5>Please begin by telling us about files you would like to upload</h5>
 							<div className='select-agency-holder'>
-								
-								{subtierAgency}
+								<div>
```

Comment:
Do we need this wrapping div around these two? Seems like an extra div that does nothing. Does the css/spacing break if you remove it?